src/Serilog/LoggerConfiguration.cs

```diff
@@ -69,6 +69,11 @@ void ApplyInheritedConfiguration(LoggerConfiguration child)
         /// </remarks>
         public LoggerAuditSinkConfiguration AuditTo => new LoggerAuditSinkConfiguration(this, s => _auditSinks.Add(s), ApplyInheritedConfiguration);
 
+        void AddOverride(string source, LoggingLevelSwitch levelSwitch)
```

Comment:
I don't know that this method is as useful now given the change in direction; inlining the assignment in the next method can save us five lines.

Reply:
yup, I'll put that back to its original state